mica-core/src/main/java/org/obiba/mica/core/domain/NetworkTable.java

```diff
@@ -1,44 +0,0 @@
-/*
- * Copyright (c) 2017 OBiBa. All rights reserved.
- *
- * This program and the accompanying materials
- * are made available under the terms of the GNU Public License v3.0.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
-package org.obiba.mica.core.domain;
-
-import java.io.Serializable;
-
-import com.google.common.base.MoreObjects;
-
-/**
- * Represents a opal {@link OpalTable} that is associated to a {@link org.obiba.mica.network.domain.Network}.
- */
-public class NetworkTable extends OpalTable implements Serializable {
```

Comment:
I'm not sure if it would be a problem. In any case, I can make sure that the tests are more explicit such that testing explicitly for either StudyTable and HarmonizationStudyTable.
I can compound this also with the fact that the method getAllOpaltables (of the HarmonizationDataset) will only return both studyTables and harmonizationStudyTables.

The only thing I see missing is an upgrade step for HarmonizationDataset where The project and table properties must be move to be a property of the linked harmonizationTable.